packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -98,6 +100,13 @@ export class StatsAnalyzer extends EventsScope {
     this.mqaSentCount = -1;
     this.lastMqaDataSent = {};
     this.lastEmittedStartStopEvent = {};
+    this.taskQueue = new TaskQueue({
+      scheduler: (task) => {
```

Comment:
why are we defining the scheduler here and not using the `defaultSchedulerFactory`?